### PR TITLE
Update Console/ClassLibrary templates to 23920

### PR DESCRIPTION
**Pull request by rchande**

### CODE REVIEW

**Comment by phenning** [comment]
You might want to hold onto this until we get a coherence-signed build from the runtime team that we can use. Also, be sure to use 2 space indentation on the project.jsons so that CTRL-K-D in vs is a noop for the json editor spacing defaults

**Comment by phenning** [comment]
FYI, I already committed a change fixing the existing spacing earlier today, so there are merge conflicts currently

**Comment by rchande** [comment]
@phenning: Thanks for the heads up, I fixed the spacing. There's also a mail thread discussing when to merge this which I'll add you to.

**Comment by Pilchie** [comment]
:+1:

File: src/BaseTemplates/ConsoleApp/project.json [hunk 1]
```diff
@@ -5,12 +5,13 @@
...
+      "type": "platform",
+      "version": "1.0.0-rc2-23930"
+    }
   },
 
   "frameworks": {
-    "netstandardapp1.5": {
-      "imports": "dnxcore50"
-    }
+    "netcoreapp1.0": {}
```
Comment (Petermarcu) [inline src/BaseTemplates/ConsoleApp/project.json]: We can't do this line yet until the rid inference change has gotten into the tooling. This doesn't work as far as I know. You can leave it as netstandardapp1.5 with no imports for now.

File: src/BaseTemplates/ClassLibrary/project.json [hunk 2]
```diff
@@ -2,12 +2,10 @@
   "version": "1.0.0-*",
```
Comment (jaredpar) [inline src/BaseTemplates/ClassLibrary/project.json]: Can we make this 1.0.0.0.  Having the `*` breaks the ability for builds to be deterministic.

File: src/BaseTemplates/ConsoleApp/project.json (hunk 1 above)
Comment (rustd) [inline src/BaseTemplates/ConsoleApp/project.json]: what is the final change for RC2?netcoreapp1.0 or netstandardapp1.5

**Pull request merged**